mpp/mpp.F90

```diff
@@ -1280,7 +1280,7 @@ module mpp_mod
 !
 ! parameter defining length of character variables
   integer, parameter :: INPUT_STR_LENGTH = 256
```

Comment:
I don't think we need to define or publicize this parameter anymore as it is no longer used within our mpp ascii routines.  I found the only use is now within our test harness.

Reply:
Fixed in latest push